WordPress/Classes/Services/ReaderSiteService.m

```diff
@@ -274,6 +262,27 @@ - (void)flagPostsFromSite:(NSNumber *)siteID asBlocked:(BOOL)blocked
     [service flagPostsFromSite:siteID asBlocked:blocked];
 }
 
+// Updates the site topic's following status in core data only.
+- (void)markUnfollowedSiteTopicWithFeedURL:(NSString *)feedURL
+{
+    ReaderSiteTopic *topic = [ReaderSiteTopic lookupWithFeedURL:feedURL inContext:self.managedObjectContext];
+    if (!topic) {
+        return;
+    }
+    topic.following = NO;
+    [[ContextManager sharedInstance] saveContext:self.managedObjectContext];
+}
+
+// Updates the site topic's following status in core data only.
+- (void)markUnfollowedSiteTopicWithSiteID:(NSNumber *)siteID
+{
```

Comment:
Nitpick, and really doesn't matter too much since these are private methods, but I "mark unfollowed site topic" didn't gave me the idea that the site topic was marked as unfollowed.

What do you think of `setSiteTopicToUnfollowedWith...` or `setSiteTopicNotFollowingWith...`?
Or even `markSiteTopicWithSiteIdToNotFollowing`, even though the argument name doesn't end up at the end of the method name 🤔